project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+class PreferenceForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cuisine: '',
+      distance: '',
+      dining_experience: '',
+      price_level: '',
+      latitude: '',
+      longitude: '',
+      open: true,
+    };
+    this.changeState = this.changeState.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  // Updates the state of the input element so it holds the chosen value.
+  changeState(event) {
+    const value =
+      event.target.name === 'open' ? event.target.checked : event.target.value;
+    this.setState({ [event.target.name]: value });
+  }
+
+  handleSubmit(event) {
+    alert('the cuisine you chose is: ' + this.state.cuisine);
+    event.preventDefault();
+  }
+
+  render() {
+    const cuisines = ['Italian', 'Mexican', 'Indian'];
+    const distances = [1, 5, 10, 25];
+    const dining_experiences = {
+      meal_takeaway: 'Takeout',
+      meal_delivery: 'Delivery',
+      restaurant: 'Eat In',
+    };
+    const prices = ['Low', 'Medium', 'High'];
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label>
+          What cuisine?
+          <select
+            class='cuisine-type'
+            name='cuisine'
+            id='cuisine'
+            onChange={this.changeState}
+            value={this.state.cuisine}
+            multiple>
+            {cuisines.map((cuisine) => (
+              <option value={cuisine}>{cuisine}</option>
+            ))}
+            ;
+          </select>
+        </label>
+        <label>
+          Distance?
+          <select
+            name='distance'
+            id='distance'
+            onChange={this.changeState}
+            value={this.state.distance}>
+            {distances.map((distance) => (
+              <option value={distance}>{distance + ' mile'}</option>
+            ))}
+            ;
+          </select>
+        </label>
+        <label>
+          Dining Experience
+          <select
+            name='dining_experience'
+            id='dining_experience'
+            onChange={this.changeState}
+            value={this.state.dining_experience}>
+            {dining_experiences.forEach((id, title) => (
+              <option value={id}>{title}</option>
+            ))}
+            ;
+          </select>
+        </label>
+        <label>
+          Price Level
+          <select
+            name='price_level'
+            id='price_level'
+            onChange={this.changeState}
+            value={this.state.price_level}>
+            {prices.map((price) => (
+              <option value={price}>{price}</option>
```

Comment:
Let's have the values for price match the numeric scale that the API uses. so like {'Low': 1, 'Medium': 2, 'High': 3}. There is also a 0 and 4, for free and very expensive, respectively, but I don't think 0 is necessary. Not sure if we want to try to include price level 4 as well...

See price_level here for more: https://developers.google.com/places/web-service/details#fields

Reply:
Lets include level 4 here, maybe add "Very High" : 4 ?